src/dsp_tools/utils/excel_to_json_resources.py

```diff
@@ -44,24 +46,62 @@ def _row2resource(row: pd.Series, excelfile: str) -> dict[str, Any]:
     """
 
     name = row["name"]
-    labels = {lang: row[lang] for lang in languages if row.get(lang)}
+    labels = {lang: row[f"label_{lang}"] for lang in languages if row.get(f"label_{lang}")}
+    if not labels:
+        labels = {lang: row[lang] for lang in languages if row.get(lang)}
     comments = {lang: row[f"comment_{lang}"] for lang in languages if row.get(f"comment_{lang}")}
     supers = [s.strip() for s in row["super"].split(",")]
 
     # load the cardinalities of this resource
-    details_df = pd.read_excel(excelfile, sheet_name=name)
+    try:
```

Comment:
If you didn't find another solution, I'm totally OK with it (also the way you did it before). I'm not sure if adding a try/catch makes it better ;)
One other way I could think of is removing all formatting from the Excel file before loading it, but you've probably already tried this?

Reply:
Is it technically possible to modify the formatting **before** parsing the file? The problem is that openpyxl is not able to open the file, so I would need another library that opens it for me, deletes the formatting, writes it back to disk, so that openpyxl can then open it...

(Or did you mean something else?)

Reply:
ideally one would use a formatting agnostic format like CSV. but I don't think we should over-complicate it at this point. It works, which is the most important thing. :)

Reply:
A short google search brought me to xlwings: https://docs.xlwings.org/en/stable/api.html
Maybe it is worth a try..?